index.d.ts

```diff
@@ -0,0 +1,75 @@
+// Type definitions for react-responsive-context
+// Project: react-responsive-context
+// Definitions by: Filipe Medeiros filipesilvamedeiros@gmail.com
+
+import { Component, FC, ReactElement, ReactNode } from "react";
+
+type BreakpointNames = 'xs' | 'sm' | 'md' | 'lg' | 'xl'
```

Comment:
This should not be a fixed constant, someone could also have "xxl" that pushes all the others to smaller resolutions

Reply:
Ignore this one, it may be like that for now, but in a future major we will rethink our defaults.

Reply:
I understand. If we wanted to make the names of the breakpoints generic (not even have more option, but be truly generic) we can do that, but I think the docs should also reflect that right? Right now the props ask for and object with these exact names, right?